addon/components/ember-tooltip-base.js

```diff
@@ -159,6 +160,19 @@ export default Component.extend({
     return `${this.get('elementId')}-wormhole`;
   }),
 
+  _fastboot: computed(function() {
```

Comment:
Any reason for not using `fastboot: service()` here (where service is `import { service } from '@ember/service';`)?

Reply:
If the `fastboot` service does not exist in the host app (i.e. in non-Fastboot apps), the injector will throw an error. This was the approach recommended by the [Addon author guide](https://www.ember-fastboot.com/docs/addon-author-guide#accessing-the-fastboot-service) for fastboot.